src/pages/EditMonitor.vue

```diff
@@ -45,11 +45,12 @@
                                         <option value="docker">
                                             {{ $t("Docker Container") }}
                                         </option>
-
+                                        <option value="local-service">
+                                            {{ $t("Local Service") }}
```

Comment:
Please name the monitor `Systemd Service` instead as it is a bit unclear what a local service is otherwise.